libraries/stencil/karma.conf.js

```diff
@@ -50,7 +50,7 @@ module.exports = function(config) {
     frameworks: ["mocha"], // use the mocha test framework
     files: [
       { pattern: path.resolve(__dirname, './node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js'), watched: false },
-      { pattern: path.resolve(__dirname, './node_modules/@webcomponents/webcomponentsjs/webcomponents-lite.js'), watched: false },
+      { pattern: path.resolve(__dirname, './node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js'), watched: false },
```

Comment:
Do you know if Stencil requires the use of the webcomponents-bundle.js?

Currently all of the other libraries on the site use webcomponents-lite.js. I know that polylfill is a bit old but we may want to update all of them at once instead of updating individual libraries.

Reply:
I don't think it requires any polyfills because it passed all the tests when the polyfills weren't being bundled properly

I made this change because I'm using the same @webcomponents/webcomponentsjs version as litElement so I just copy pasta this from that 😆

Reply:
oh ok, i didn't realize litElement was already using it.